cves/kernel/CVE-2012-3520.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: It was discovered that Linux would accept spoofed messages.
```

Comment:
You should be more specific about the exact bug, "Linux would accept spoofed messages" doesn't give a lot of information